src/pool/srv_pool.c

```diff
@@ -2102,6 +2103,14 @@ ds_pool_connect_handler(crt_rpc_t *rpc)
 		D_GOTO(out_map_version, rc);
 	}
 
+	rc = ds_sec_cred_get_origin(&in->pci_cred, &machine);
```

Comment:
in the "check existing pool handles" code above is there a scenario in which, if it were to be checked, the origin (machine name) associated with this connect request may not match the machine name in the existing pool handle found (by its handle UUID value)?

Reply:
Its possible maybe if you did a connect on a handle that was collectively opened? Under what circumstances does this happen? I was under the impression that a pool handle included RDMA transfer information so it was specific to a given machine.